src/components/button/button.tsx

```diff
@@ -59,11 +64,15 @@ const Button: React.FunctionComponent<ButtonProps> = props => {
 	const classes = classNames(prefixCls, className, {
 		[`${prefixCls}-${sizeCls}`]: sizeCls,
 		[`${prefixCls}-${type}`]: type,
+		[`${prefixCls}-loading`]: loading,
 	})
 
+	const iconNode = loading ? 'loading' : null
+
 	return (
 		<button type={htmlType} className={classes} onClick={handleClick} {...rest}>
 			{children}
+			{iconNode}
```

Comment:
Should this be rendered here? Since if the button is in loading state, both button text, icon and the "loading" text will be rendered at the same time.
I think it should be:
`<button>{loading ? 'loading' : children}</button>`